src/core/server/root/index.ts

```diff
@@ -98,10 +94,11 @@ export class Root {
   }
 
   private async setupLogging() {
+    const { configService } = this.server;
```

Comment:
__note:__ it's becoming clearer and clearer that we may need to merge `server` with `root` someday :)